pkg/internal/export/otel/common.go

```diff
@@ -51,14 +51,19 @@ var DefaultBuckets = Buckets{
 }
 
 func otelResource(service svc.ID) *resource.Resource {
+	telemetrySDK := semconv.TelemetrySDKLanguageGo
+
+	if service.Technology != "" {
```

Comment:
Given that the OTEL key is "telemetry.sdk.language", shall we change the `Technology` field name by something similar to `SDKLanguage`, `Lang`, `Language`...?

Not a strong opinion. If you feel `Technology` is more accurate, let's keep it.